wp-content/themes/citylimits/homepages/assets/less/homepage.less

```diff
@@ -224,7 +248,7 @@
       line-height: 2rem;
 
       border-right: 1px solid @grey1;
-      padding-right: @measure;
+      padding-right: @measure + 1;
```

Comment:
Note: LESS calculates this as `1.5rem + 1 = 2.5rem`

To avoid future compilation problems if LESS decides to require units,  if `1` is meant to be a number of `rem` then it should have a unit.

Reply:
Good catch. Updated in https://github.com/INN/umbrella-citylimits/pull/113/commits/493bf9f0a94f479b553c58b38fcd3ec2601d6edb